### PR TITLE
Issue #5 - Upgrade to 5.8

### PR DESCRIPTION
**Pull request by hfiguiere**
Removed upstreamed patches. Updated screenshot links (the others were 404)

### CODE REVIEW

File: com.rawtherapee.RawTherapee.yaml [hunk 1]
```diff
@@ -230,33 +230,11 @@ modules:
       - -DCMAKE_BUILD_TYPE=RelWithDebInfo
     sources:
       - type: archive
-        url: https://github.com/Beep6581/RawTherapee/releases/download/5.7/rawtherapee-5.7.tar.xz
-        sha256: dbd7c7cf7488fb97c520821eee2c745291637644b391e3ec0ed3a29701f1a9c7
-      # Update XDG files
+        url: https://github.com/Beep6581/RawTherapee/releases/download/5.8/rawtherapee-5.8.tar.xz
+        sha256: 360528a0aae922eb5af8742f415475fb91b6d62a739da5f2804828f04ec40853
       # Update screenshots in appdata file
-      # Add developer_name tag
```
Comment (scx) [inline com.rawtherapee.RawTherapee.yaml:232]: It isn't upstreamed.

**Review by scx — CHANGES_REQUESTED** [review]
Some patches have not been upstreamed. We must keep them.

File: com.rawtherapee.RawTherapee.yaml (hunk 1 above)
Comment (hfiguiere) [reply com.rawtherapee.RawTherapee.yaml:232]: this is why one patch per patch file is nice. Because we see better.

**Comment by hfiguiere** [comment]
Also for reference, I reverted the appdata patch: when I submitted it to change the screenshots, upstream re-uploaded the missing images on their website instead. So nothing needs to change.

**Comment by scx** [comment]
@hfiguiere 

> Also for reference, I reverted the appdata patch: when I submitted it to change the screenshots, upstream re-uploaded the missing images on their website instead. So nothing needs to change.

This is the reason why I prefer to wait a couple days after release. Usually, various fixes appear within a week of release.

As for the patch for AppData file, you can find it here:
https://github.com/Beep6581/RawTherapee/commit/4943af4743b3b6048f99fa56c858cd8585405d67

However, some time ago I've decided that it would be easier to use `xmlstarlet`.
https://github.com/scx/rawtherapee-flatpak/commit/634481faaa054bbd9c34852772080616dc637579#diff-060b4052b990b0008f869f319e4dcf21L57-R61

Anyway, I made another pull request in which I updated gtkmm and added a patch for installing additional icons.
You can backport these changes or I'll merge my PR.
https://github.com/flathub/com.rawtherapee.RawTherapee/pull/7

**Pull request closed without merging**